src/models/user.ts

```diff
@@ -81,6 +82,15 @@ const UserSchema = new mongoose.Schema({
     type: Number,
     default: 1500,
   },
+  ratingDeviation: {
+    type: Number,
+    default: 350,
+  },
+  ratingVolatility: {
+    type: Number,
+    default: 0.06,
+  },
+
```

Comment:
Let's remove these changes for now. We don't necessarily want these going into master just yet!